### PR TITLE
Covid-19 World Wide

### PR DESCRIPTION
**Pull request by Mucyosoda**
**Covid19 app**
This project provides all information numerically in each country, the first-page displays all the countries, when you click on one country the other page with country info will display. the page also includes a search where you can search by country name.
Countries and details are gotten and stored in the API

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Mucyosoda,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
-# covid-19-world-wide
-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Covid-19 World Wide
+
+This project provides all information numerically in each country, the first-page displays all the countries, when you click on one country the other page with country info will display. the page also includes a search where you can search by country name.
```
Comment (mikenath223) [inline README.md:5]: -  [ ] In the readme kindly include instructions to get the project started.

File: src/__Test__/componenttest/Nav.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+          <Route path="/country">
+            <Navbar />
+          </Route>
+
+        </HashRouter>
+        ,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
```
Comment (mikenath223) [inline src/__Test__/componenttest/Nav.test.js:21]: - [ ] There are a couple of failing snapshot tests please update them.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Mucyosoda , 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**